roles/download/defaults/main.yml

```diff
@@ -13,11 +13,13 @@ etcd_download_url: "https://github.com/coreos/etcd/releases/download/{{ etcd_ver
 calico_download_url: "https://github.com/Metaswitch/calico-docker/releases/download/{{calico_version}}/calicoctl"
 calico_cni_download_url: "https://github.com/projectcalico/calico-cni/releases/download/{{calico_cni_version}}/calico"
 calico_cni_ipam_download_url: "https://github.com/projectcalico/calico-cni/releases/download/{{calico_cni_version}}/calico-ipam"
+weave_download_url: "http://git.io/weave"
```

Comment:
If you are checksumming the file, you should use a versioned URL also `https://github.com/weaveworks/weave/releases/download/v1.4.4/weave`